scss/book-list-detail.scss

```diff
@@ -0,0 +1,60 @@
+.body{
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
This is the same for every page, it should be in a `_commons.scss` or in the `application.scss`